touchforms/formplayer/static/formplayer/script/test_vendor/underscore.js

```diff
@@ -0,0 +1,1548 @@
+//     Underscore.js 1.8.3
```

Comment:
I'm not sure if this is only for tests, or if it's used in any other context (looks like only tests). [commcare-hq](https://github.com/dimagi/commcare-hq) currently contains at least (from a quick search) Underscore 1.3.1, 1.4.4 and 1.6.0, and Vellum uses 1.5.2. All of them are pretty old and contain backward-incompatible APIs with 1.8.x. This is a pain for Vellum dev, but I haven't tried to tackle it yet as it seems to be a pretty nasty problem to bring all API use up to the latest version. And maybe we want to consider switching to lodash? Anyway, this is mainly just FYI. Not sure if there is anything actionable other than it could be confusing to have different versions in different parts of the codebase, especially given that there are backward-incompatible changes in newer versions.